src/Tizen.NUI/src/public/Layouting/LayoutItem.cs

```diff
@@ -203,10 +203,16 @@ internal void AttachToOwner(View owner)
         /// <since_tizen> 6 </since_tizen>
         public void Measure(MeasureSpecification widthMeasureSpec, MeasureSpecification heightMeasureSpec)
         {
-            OnMeasure(widthMeasureSpec, heightMeasureSpec);
-            OnMeasureIndependentChildren(widthMeasureSpec, heightMeasureSpec);
-            flags = flags | LayoutFlags.LayoutRequired;
-            flags &= ~LayoutFlags.ForceLayout;
+            bool needsLayout = NeedsLayout(widthMeasureSpec.Size.AsDecimal(), heightMeasureSpec.Size.AsDecimal(), widthMeasureSpec.Mode, heightMeasureSpec.Mode);
+
+            needsLayout = needsLayout || ((flags & LayoutFlags.ForceLayout) == LayoutFlags.ForceLayout);
```

Comment:
This line is not required because NeedsLayout checks this line already.

Reply:
I added it here (took it form the old code before optimization) because it does not exists in the NeedsLayout function !
I'm not familiar with this part if code, so this check is needed in NeedsLayout function ? should I move it there ?

Reply:
In NeedsLayout(), LayoutRequested is checked.
LayoutRequested getter is as follows.
```
get
{
    return (flags & LayoutFlags.ForceLayout) == LayoutFlags.ForceLayout;
}
```

Therefore, you don't need to add the code if you call NeedsLayout().

Reply:
thank you